### DISCUSSION TITLE
I cannot connect to mongo db How do i connect??

### DISCUSSION THREAD

**Discussion by bytesagar** [prisma/prisma#12065] [Q&A] [answered]
Error in connector: Database error. error code: unknown, error message: Transactions are not supported by this deployment
    at cb (/home/bytesagar/starter/node_modules/@prisma/client/runtime/index.js:36067:17)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  clientVersion: '2.30.2'

**Comment by janpio** [comment, marked as answer]
https://www.prisma.io/docs/concepts/database-connectors/mongodb#error-transactions-are-not-supported-by-this-deployment

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
